examples/notebooks/cache_dataset_speed.ipynb

```diff
@@ -60,11 +60,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "data_root = '/workspace/data/medical/Task09_Spleen'\n",
+    "data_root = '/Users/johnsonhj/data/Task09_Spleen'\n",
```

Comment:
Suggest keeping this original `cache_dataset_speed` notebook and add another `persist_dataset_speed` for your case. Thanks.